src/sbvr-api/permissions.ts

```diff
@@ -862,9 +862,7 @@ const rewriteRelationship = memoizeWeak(
 								foundCanAccessLink = true;
 							}
 							// return a true expression to not select the relationship, which might be virtual
-							// this should be a boolean expression, but needs to be a subquery in case it
-							// is wrapped in an `or` or `and`
-							return ['Equals', ['Boolean', true], ['Boolean', true]];
+							return ['Boolean', true];
```

Comment:
Just curious, which changes was that allowed this?

Reply:
It's allowed by using true booleans, ie `['Boolean', true]` will now generate `TRUE` rather than `1` which means `WHERE TRUE` is valid vs `WHERE 1` which is not